pam/integration-tests/integration_test.go

```diff
@@ -239,6 +253,22 @@ func saveArtifactsForDebug(t *testing.T, artifacts []string) {
 	}
 }
 
+func prepareCLILogging(t *testing.T) string {
+	t.Helper()
+
+	cliLog := filepath.Join(t.TempDir(), "authd-pam-cli.log")
+	t.Cleanup(func() {
+		out, err := os.ReadFile(cliLog)
+		if os.IsNotExist(err) {
+			return
+		}
```

Comment:
IIRC, the `os.IsNotExist` error is an alias for `fs.ErrNotExist` and they themselves recommend comparing the error with `errors.Is(err, fs.ErrNotExist)`. Would you mind switching to this format?